src/super_gradients/training/models/detection_models/yolov5_base.py

```diff
@@ -414,7 +423,7 @@ def _initialize_weights(self):
             elif t is nn.BatchNorm2d:
                 m.eps = 1e-3
                 m.momentum = 0.03
-            elif t in [nn.LeakyReLU, nn.ReLU, nn.ReLU6, nn.Hardswish]:
+            elif t in [nn.LeakyReLU, nn.ReLU, nn.ReLU6, nn.Hardswish, nn.SiLU]:
```

Comment:
is this related to the depthwise change somehow? or a separate bugfix?

Reply:
Separate indeed. I was just comparing the Nano model definition in our codebase vs the source and noticed that SiLU isn't inplace, unlike in the original implementation